kaggle/RANZCR/4th_place_solution/utils.py

```diff
@@ -0,0 +1,116 @@
+import os
+import random
+
+import numpy as np
+import torch
+from monai.optimizers.lr_scheduler import WarmupCosineSchedule
+from numba import jit
+from torch import optim
+from torch.utils.data import DataLoader, SequentialSampler
+
+from data.seg_data import CustomDataset
+
+
+def set_seed(seed=1234):
+    random.seed(seed)
+    os.environ["PYTHONHASHSEED"] = str(seed)
```

Comment:
looks like this line of setting PYTHONHASHSEED has no effect, not sure whether we really need it ([ref](https://stackoverflow.com/questions/25684349/can-os-environpythonhashseed-be-set-dynamically-from-within-an-application))